mycroft/util/lang/parse_en.py

```diff
@@ -1377,7 +1365,7 @@ def date_found():
             idx += used - 1
             found = True
     # check that we found a date
-    if not date_found:
+    if not date_found():
```

Comment:
This was identified earlier (see PR #2300) and is waiting for @krisgesling to clear. Some skills were expecting this to not function.

I suggest you remove this change for the time being so we can go ahead and review / merge the other changes

Reply:
force-pushed.